docs/fontbakery.js

```diff
@@ -0,0 +1,281 @@
+  const tabid = $('#v-pills-tab').children().length;
+  const result = data.get('result');
+  const checkid = data.get('key');
+  let thispill = $(`#v-pills-tab button[data-checkid="${checkid}"]`);
+  // console.log("Adding result for ",checkid)
+  if (thispill.length == 0) {
+    // Add a new pill
+    thispill = $(`
+        <button
+          class="nav-link bg-${result}"
+          id="v-pills-${tabid}-tab"
+          data-toggle="pill"
+          data-target="#v-pills-${tabid}"
+          data-sortorder="${SORT_RESULT[result]}"
+          type="button"
+          role="tab"
+          data-checkid=${data.get('key')}
+          aria-controls="v-pills-${tabid}">${data.get('description')}</button>
+      `);
+    $('#v-pills-tab').append(thispill);
+  }
+  let thistab = $(`#v-pills-tabContent div[data-checkid="${checkid}"]`);
+  if (thistab.length == 0) {
+    thistab = $(`
+      <div
+        class="tab-pane fade"
+        data-sortorder="${SORT_RESULT[result]}"
+        id="v-pills-${tabid}"
+        role="tabpanel"
+        aria-labelledby="v-pills-${tabid}-tab"
+        data-checkid=${data.get('key')}
+      >
+        <h4>${data.get('description')}</h4>
+        <p class="text-muted">${data.get('key')}</p>
+        <div class="rationale">
+        ${ CmarkGFM.convert((data.get('rationale')||'').replace(/^ +/gm, '')) }
+        </div>
+        <ul class="results">
+        </ul>
+      </div>
+      `);
+    $('#v-pills-tabContent').append(thistab);
+  }
+  // Update pill / tab results with worst result
+  if (SORT_RESULT[result] < thispill.data('sortorder')) {
+    thispill.removeClass(function(index, className) {
+      return (className.match(/(^|\s)bg-\S+/g) || []).join(' ');
+    });
+    thispill.addClass('bg-' + result);
+    thispill.data('sortorder', SORT_RESULT[result]);
+    thistab.data('sortorder', SORT_RESULT[result]);
+  }
+
+  for (log of data.get('logs')) {
+    let where = 'ul.results';
+    if (data.has('filename')) {
+      const filename = data.get('filename');
+      where = `ul.results li ul[data-filename='${filename}']`;
+      if (thistab.find(where).length == 0) {
+        thistab.find('ul.results').append(`<li>
+          <b>${filename}</b>
+          <ul data-filename="${filename}">
+          </ul>
+        </li>`);
+      }
+    }
+    thistab.find(where).append($(`
+          <li>
+            <span
+              class="bg-${log.get('status')} font-weight-bold">
+              ${log.get('status')}
+            </span>:
+            <div>${CmarkGFM.convert(log.get('message'))}</div>
+          </li>
+        `));
+  }
+  // Sort the tabs based on result
+  tinysort('div#v-pills-tab>button', {'data': 'sortorder'});
+  tinysort('div#v-pills-tabContent>div', {'data': 'sortorder'});
+}
+
+/* Add a profile from the profiles list */
+PROFILES = {
+  'opentype': 'OpenType (standards compliance)',
+  'universal': 'Universal (community best practices)',
+  'googlefonts': 'Google Fonts (for submission to Google)',
```

Comment:
please remove "(for submission to Google)", as it is redundant and a similar message is not used for the other vendor-specific profiles